ome/omero-insight
# Gradle build version of insight

Requires testing on multiple OS's and on a server account with plenty of data.

Prerequisites to test:

1. install gradle on your system (via homebrew if mac, via sdkman if linux)

Build and run insight:

1. `cd omero-insight`
2. `gradle run`
3. Check that insight works as before, no icons missing etc.

Build insight:
1. `cd omero-insight`
2. `gradle build`
3. A build directory should be visible
4. Go to the `distributions` folder
5. Unzip `omero-insight.zip`
6. `cd omero-insight`
7. Run `bin/omero-insight`
8. Now run `bin/omero-insight containerImporter.xml`. Check that the importer starts

insight-ij
1. Unzip `omero-insight-imagej.zip`
2. Copy the directory to `Fiji/plugins`
3. Check that you can connect to OMERO.
